core/src/main/java/org/frankframework/pipes/ForEachChildElementPipe.java

```diff
@@ -300,7 +300,7 @@ private void createHandler(HandlerRecord result, ThreadConnector<?> threadConnec
 		if (getXmlDebugger()!=null && (StringUtils.isNotEmpty(getContainerElement()) || StringUtils.isNotEmpty(getTargetElement()) || getExtractElementsTp()!=null)) {
 			String containerElementString = StringUtils.isNotEmpty(getContainerElement()) ? "filter to containerElement '"+getContainerElement()+"'" : null;
 			String targetElementString = StringUtils.isNotEmpty(getTargetElement()) ? "filter to targetElement '"+getTargetElement()+"'" :null;
-			String xpathString = getExtractElementsTp()!=null ? "filter XPath '"+getElementXPathExpression()+"'": null;
+			String xpathString = getExtractElementsTp()!=null ? "filter XPath (with silent root element)'"+getElementXPathExpression()+"'": null;
```

Comment:
Wat bedoel je met "silent" root element?
Ik vraag me af of daar niet een betere bewoording voor is. `(with root element named 'root')` misschien?

(Ik wou trouwens ooit nog die root-element name configurable maken, was een triviale aanpassing aan de code geloof ik).

Reply:
is temporary beter?